deploy/csi-rclone/templates/csi-nodeplugin-rclone.yaml

```diff
@@ -154,6 +156,10 @@ spec:
         {{ toYaml . | nindent 8 }}
     {{- end }}
       volumes:
+      - hostPath:
+          path: /tmp/{{.Release.Namespace}}-{{.Release.Name}}-{{.Release.Revision}}
+          type: DirectoryOrCreate
+        name: node-temp-dir
```

Comment:
```suggestion
      - hostPath:
          # NOTE: We mount on /tmp because we want the saved configuration to not survive a whole node restart.
          path: /tmp/{{.Release.Namespace}}-{{.Release.Name}}-{{.Release.Revision}}
          type: DirectoryOrCreate
        name: node-temp-dir
```

Reply:
It is good to note that the /tmp path is deliberate. Feel free to reword as you see fit. But the comment is useful.